addons/calendar_sms/models/calendar.py

```diff
@@ -42,9 +42,14 @@ def get_next_mail(self):
         """ Cron method, overridden here to send SMS reminders as well
         """
         result = super(AlarmManager, self).get_next_mail()
+
+        cron = self.env.ref('calendar.ir_cron_scheduler_alarm', raise_if_not_found=False)
+        if not cron:
+            _logger.error("Cron for " + self._name + " can not be identified !")
+            return False
```

Comment:
Is it useful to change the behaviour there? The old version would blow up if the cron was not found. Indeed since it's the `ir_cron_scheduler_alarm` which calls this method, there'd be something really odd if it were invoked without the caller cron existing.

Reply:
Well, the super method is implemented like this, and supports the case when the cron doesn't exist anymore, I thought it would be more coherent to do the same here.
https://github.com/odoo/odoo/blob/d1caa5c6afe80ee421fded7b4c82bf5cf9690c14/addons/calendar/models/calendar.py#L366-L369